backend/apis/clients/list-clients.api.js

```diff
@@ -1,23 +1,42 @@
-const { app, databaseError, pool, invalidRequiest } = require("../../server");
+const { app, databaseError, pool, invalidRequest } = require("../../server");
 const mysql = require("mysql");
 
 app.get("/api/clients", (req, res, next) => {
   pool.getConnection((err, connection) => {
     if (err) {
       return databaseError(req, res, err);
     }
-    var queryString = "SELECT ? AS result";
 
-    const query = mysql.format(queryString, [req.query.foo]);
+    var queryString =
+      "SELECT cl.id,cl.firstName,cl.lastName,cl.birthday,ct.zip,ct.primaryPhone,cl.addedBy as addedById,us.firstName as addedByFirstName,us.lastname as addedByLastName,cl.dateAdded FROM clients cl JOIN contactInformation ct ON cl.id = ct.clientId JOIN users us ON cl.addedBy = us.Id WHERE (cl.firstName LIKE ? AND cl.lastName LIKE ?) OR ct.zip = ? ORDER BY cl.firstName,cl.lastName DESC LIMIT 100";
```

Comment:
This query would be easier to understand if the indentation were better. You can use [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to do multiline strings in javascript.

For example:
```js
const query = `
  SELECT cl.id, cl.firstName, cl.birthday, ct.zip, ct.primaryPhone, cl.addedBy as addedBy,
    us.firstName as addedByFirstName, us.lastName as addedByLastName, cl.dateAdded
  FROM
    clients cl
  JOIN
    contactInformation ct ON cl.id = ct.clientId
  JOIN
    users us on cl.addedBy = us.id
  WHERE
    (cl.firstName LIKE ? AND cl.lastName LIKE ?) OR ct.zip = ?
  ORDER BY cl.firstName, cl.lastName DESC LIMIT 100
`
```

In writing that, I noticed that you have `us.Id` instead of `us.id` -- we should fix that.

Reply:
Got it..